src/services/i18n/I18n.ts

```diff
@@ -4,6 +4,12 @@ import type Env from '@/services/env/Env'
 import { camelCaseToWords } from '@/utilities/camelCaseToWords'
 import { get } from '@/utilities/get'
 
+declare module 'intl-messageformat' {
+  interface Options {
+    fallback: string
+  }
```

Comment:
I keep seeing the term `defaultMessage` being used for this, maybe we should use that name?

https://formatjs.io/docs/react-intl/components/#message-descriptor
https://formatjs.io/docs/vue-intl/#using-icu-in-vue-sfc